kalitectl.py

```diff
@@ -609,6 +628,8 @@ def collect_mem_usage(_sig, _frame):
     arguments = docopt(__doc__, version=str(VERSION), options_first=True)
 
     if arguments['start']:
+        if arguments["--port"]:
```

Comment:
You should do this no matter the command! All commands can potentially make use of a custom --port argument